### DISCUSSION TITLE
FEATURE REQUEST: scene names

### DISCUSSION THREAD

**Discussion by DoodlerPete** [keiwando/toonsquid#284] [Ideas]
I would love the feature to change the scene names. If there is no vector symbol sharing between different projects. I could use different scenes and use the symbols in different scenes. 
thanks
Pete.

**Comment by Cat-Butler** [comment]
Already available, although not obvious.
Tap on the current scene name to bring up the list of scenes.
Swipe from right to left on any scene name in the list to reveal the options to Rename, Duplicate, or Delete a scene.
